config/routes.rb

```diff
@@ -3,10 +3,12 @@
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
-  root 'neighborhoods#index'
+  resources :neighborhoods
+  #root 'neighborhoods#index'
+  root :to => redirect('/neighborhoods')
```

Comment:
Hi Bryan, the routes look good and I follow everything except for line 8.  Why did you change 
-root 'neighborhoods#index' 
to 
root :to => redirect('/neighborhoods')

Is this better practices somehow to redirect?  Let me know, thanks!